addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/internal/util/NetworkUtil.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.globalcache.internal.util;
+
+import java.net.Inet6Address;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.Enumeration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class to retrieve local network interfaces.
+ *
+ * @author Mark Hilbush
+ */
+public class NetworkUtil {
```

Comment:
as this is nothing specific to your binding, you might consider moving/merging this method into org.eclipse.smarthome.core.net.NetUtil